code/game/objects/machinery/vending/marine_vending.dm

```diff
@@ -34,6 +34,9 @@
 			/obj/item/ammo_magazine/smg/standard_smg = -1,
 			/obj/item/weapon/gun/smg/standard_machinepistol = -1,
 			/obj/item/ammo_magazine/smg/standard_machinepistol = -1,
+			/obj/item/weapon/gun/smg/standard_stormsmg = -1,
+			/obj/item/ammo_magazine/smg/standard_stormsmg = -1,
+			/obj/item/ammo_magazine/smg/standard_stormsmg/drum = 4,
```

Comment:
either both mags are balanced and free, or only one exist. Not that

Reply:
Why?

Reply:
because it will get slotted into imports